### ISSUE TITLE
Minecraft pistons cause game crash

### ISSUE THREAD

**Issue by Pyxial** [embeddedt/ModernFix#324]
I'll add a video if needed

TL;DR Pistons crash the game, I don't think its an issue with mod confliction, it also produces a side effect that is probably pretty bad :/

So, I was adding any mod I see to my mod pack, as you do, and I found this one so I added it. After adding it, the game would crash every time a piston moved (checking logs it mentions that the model is missing, but I didn't check it at first)

It could very well be an issue with this mod and another, Not Enough Crashes says this mod, more culling, and nbt auto complete are causing the crash. I doubt nbt auto complete is the issue, and more culling only hides stuff I'm not looking at, yet I'm looking at the pistons so idk.

After removing the mod from the server (in this case client because java hosts single player on a local server) all pistons that have been involved in a crash no longer move while facing the same direction. You can make the piston face a different way, but the piston won't push the way it was originally facing even after you remove the mod

**Comment by embeddedt** [comment]
Please post a `latest.log` file. It is likely the same bug in More Culling that a lot of people reported a while ago.

**Comment by Pyxial** [comment]
Ok, also, using the visible barriers mod I found that it gets replaced with block 36, which stops the pistons from extending. Also I guess it's not the model missing

Mod loader logs: [removed]

[13:38:14] [Thread-26/ERROR]:
 Block entity "minecraft:piston" constructor thrown error: java.lang.ExceptionInInitializerError
[13:38:14] [Thread-26/ERROR]:
 Unable to get block entity class for "minecraft:piston"

[removed]

**Comment by Pyxial** [comment]
oops, closed it :/

**Comment by embeddedt** [comment]
There isn't a crash in that `latest.log` file.

**Comment by Pyxial** [comment]
huh, this might work, but idk

[removed]

**Comment by Pyxial** [comment]
[removed]

or this

**Comment by embeddedt** [comment]
Does the issue occur if More Culling is removed?

The block 36 issue is probably just a side effect of the game crashing and therefore saving the world in an inconsistent state.

**Comment by Pyxial** [comment]
Yes, when more culling is removed I am able to use pistons without crashing. Now I just gotta find out why a chat mod is included in the NEC error.

**Comment by embeddedt** [comment]
It's probably the original issue (https://github.com/FxMorin/MoreCulling/issues/195) then, which I can't fix on my end.

> Now I just gotta find out why a chat mod is included in the NEC error.

I wouldn't worry about this if the game is working fine; NEC is often incorrect about which mod was really the problem, especially when multiple are listed.
